frontend/src/app/app.component.html

```diff
@@ -1 +1,7 @@
-<router-outlet></router-outlet>
+<volontulo-header></volontulo-header>
+<div class="container">
```

Comment:
I don't know if it should be here, as on homepage in this part of page is a baner, that spread on the whole width.